tests/integration/targets/mso_schema_site_anp/tasks/main.yml

```diff
@@ -390,42 +277,6 @@
     - nm_query_non_anp is not changed
     - nm_query_non_anp.msg == "ANP 'non_existing_anp' not found"
 
-# USE A NON-EXISTING STATE
-- name: Non-existing state (normal_mode)
-  cisco.mso.mso_schema_site_anp:
-    <<: *mso_info
-    schema: '{{ mso_schema | default("ansible_test") }}'
-    site: 'aws_{{ mso_site | default("ansible_test") }}'
-    template: Template 1
-    anp: ANP
-    state: non-existing-state
-  ignore_errors: true
-  register: nm_non_existing_state
-
-- name: Verify non_existing_state
-  ansible.builtin.assert:
-    that:
-    - nm_non_existing_state is not changed
-    - nm_non_existing_state.msg == "value of state must be one of{{':'}} absent, present, query, got{{':'}} non-existing-state"
-
-# USE A NON-EXISTING SCHEMA
-- name: Non-existing schema (normal_mode)
-  cisco.mso.mso_schema_site_anp:
-    <<: *mso_info
-    schema: non-existing-schema
-    site: 'aws_{{ mso_site | default("ansible_test") }}'
-    template: Template 1
-    anp: ANP
-    state: query
-  ignore_errors: true
-  register: nm_non_existing_schema
-
-- name: Verify non_existing_schema
-  ansible.builtin.assert:
-    that:
-    - nm_non_existing_schema is not changed
-    - nm_non_existing_schema.msg == "Provided schema 'non-existing-schema' does not exist."
-
```

Comment:
Should we move those tests to on-prem site vs removing them?

Reply:
Added back `USE A NON-EXISTING STATE` task. The other one is already part of the `on-prem` task.